reference-architectures/deploy-arch-ibm-vsi-fully-configurable.md

```diff
@@ -0,0 +1,167 @@
+
+# The release that the reference architecture describes
+version: 5.8.1
+
+# Whether the reference architecture is published to Cloud Docs production.
+# When set to false, the file is available only in staging. Default is false.
+production: false
+
+# Use if the reference architecture has deployable code.
+# Value is the URL to land the user in the IBM Cloud catalog details page
+# for the deployable architecture.
+# See https://test.cloud.ibm.com/docs/get-coding?topic=get-coding-deploy-button
+deployment-url: https://cloud.ibm.com/catalog/architecture/deploy-arch-ibm-slz-vsi-ef663980-4c71-4fac-af4f-4a510a9bcf68-global
+
+docs: https://cloud.ibm.com/docs/secure-infrastructure-vpc
+
+image_source: https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone-vsi/blob/main/reference-architectures/vsi.svg
+
+related_links:
+  - title: "Landing zone for applications with virtual servers - Standard - Integrated setup with configurable services"
+    url: "https://cloud.ibm.com/docs/deployable-reference-architectures?topic=deployable-reference-architectures-lz-vsi-fully-configurable-ra"
+    description: "A comprehensive landing zone deployable architecture that provides virtual servers in a secure VPC for enterprise workloads."
+
+use-case: Cybersecurity
+industry: Banking,FinancialSector
+compliance: FedRAMP
+
+content-type: reference-architecture
+
+---
+
+{{site.data.keyword.attribute-definition-list}}
+
+# Landing zone for applications with virtual servers - Standard - Integrated setup with configurable services variation
+{: #lz-vsi-fully-configurable-ra}
+{: toc-content-type="reference-architecture"}
+{: toc-industry="Banking,FinancialSector"}
+{: toc-use-case="Cybersecurity"}
+{: toc-compliance="FedRAMP"}
+{: toc-version="5.8.1"}
+
+The Standard - Integrated setup with configurable services variation of the Landing zone for applications with virtual servers deployable architecture provides a flexible and comprehensive solution for deploying virtual servers with extensive customization options. This architecture supports multiple VSI configurations, load balancers, block storage volumes, and advanced networking features across multiple subnets and availability zones.
+
+This variation is best suited for production environments and enterprise workloads that require advanced configurations, high availability across multiple zones, comprehensive security controls, and extensive customization options for networking, storage, and load balancing.
+
+## Architecture diagram
+{: #ra-vsi-fc-architecture-diagram}
+
+![Architecture diagram for the Standard variation of Landing zone for applications with virtual servers](vsi.svg "Architecture diagram for the Standard - Integrated setup with configurable services variation of the Landing zone for applications with virtual servers deployable architecture"){: caption="Standard variation of Landing zone for applications with virtual servers" caption-side="bottom"}{: external download="vsi.svg"}
```

Comment:
There are two standard variations! This one is called "Standard - Integrated setup with configurable services"